index.php

```diff
@@ -155,29 +153,36 @@ function ConnectionLost(res) {
           }
         }
 
-        var listCounter = [0,0];
+        var listCounter = 30;
         setInterval(function () {
-            listCounter[0]++;
-            listCounter[1]++;
+            listCounter++;
         }, 1000);
 
         /*Callback for incoming message processing */
         function MessageArrived(message) {
-          var type = message.destinationName.split("/")[1] == "temperature" ? 0 : 1
+          if (listCounter < 30) { return ; }
+
           var date = new Date()
+          writeChart(message.payloadString, date);
+
+          listCounter = 0;
+          tempChart.update(), umidChart.update();
+        }
 
-          if (listCounter[type] < 30) { return ; }
+        function writeChart(msg, date) {
```

Comment:
```suggestion
        function writeChart(msg) {
```
Não faz sentido passar date se ele não é utilizado na função

Reply:
Verídico, esqueci de tirar